src/clients/c++/examples/CMakeLists.txt

```diff
@@ -63,6 +63,9 @@ install(
 # ensemble_image_client
 #
 add_executable(ensemble_image_client ensemble_image_client.cc)
+if(${TRTIS_ENABLE_GPU})
```

Comment:
You shouldn't need these. request library rule already adds the target include so it should be inherited.